server/models/procedures/invoicing.sql

```diff
@@ -631,10 +631,9 @@ BEGIN
     SELECT payment.uuid, payment.balance FROM stage_payment_balances AS payment;
   DECLARE CONTINUE HANDLER FOR NOT FOUND SET done = TRUE;
 
-  -- This is marked as a "transfer" transaction type.
   -- TODO(@jniles): propose issue for how to deal with this kind of transaction
   -- type that is automatically generated by the system.
-  SET linkTransactionTypeId = 5;
+  SET linkTransactionTypeId = 19;
```

Comment:
What is meaning of these numbers ? 19 instead of 5

Reply:
Ah, it's the new transaction type ID for caution links.  I supposed I should have named the variable "cautionLinkTransactionTypeId", since "link" could be anything... :/